test/unit/world_location_test.rb

```diff
@@ -211,7 +211,7 @@ class WorldLocationTest < ActiveSupport::TestCase
     should "allow accessing required top-level attributes" do
```

Comment:
@pmanrubia I have made changes that no longer require defining the slug method anymore..

See https://github.com/alphagov/smart-answers/blob/3c8ff7a363617d959145b70ac58a7d934757b1ec/app/models/world_location.rb#L61